File: C++/1480. Running Sum of 1d Array.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int> solution(nums.size());
        solution[0] = nums[0];
        for(int i = 1; i < nums.size(); i++){
            solution.at(i) = nums.at(i) + solution[i - 1];
        }
        return solution;
    }

};

File: Python/1480. Running Sum of 1d Array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        solution = []
        solution.append(nums[0])
        for i in range(1, len(nums)):
            solution.append(solution[i - 1] + nums[i])
        return solution